include/libopencm3/stm32/g4/adc.h

```diff
@@ -537,8 +538,8 @@
 #define ADC_CCR_DUAL_ALTERNATE_TRIG			0x9
 /**@}*/
 
-#define ADC_CCR_DUAL_MASK				(0x1f)
 #define ADC_CCR_DUAL_SHIFT				0
+#define ADC_CCR_DUAL_MASK				(0x1f << ADC_CCR_DUAL_SHIFT)
```

Comment:
see, this is exactly why having these mixed styles causes problems :)

Reply:
A better question would be how both shifted and unshifted styles ended up in that file **at the same time**. :)

Reply:
that's easy, people were demanding faster turn arounds, they swore up and down that it was reviewed, they got their friends to swear off, and presto, it wasn't actually.  It was a hacked up copy/paste of another family, with a few things hastily done.  There are _reasons_ why I don't "just merge" people's changes. This is just one of them.

Reply:
No doubt there are reasons! I'd never vote for anything like "just merge" for goodness' sake, no! I'm all in for reasonable scrutiny and consistent approach. I hope you'll agree that changes hanging in a limbo have nothing to do with that though...